### PR TITLE
chore(doc): add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by mkxbl**
**What type of PR is this?**

chore(doc)

**What this PR does / why we need it**:

add CONTRIBUTING.md

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+## Code of Conduct
+
+### Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
```
Comment (KaoImin) [inline CONTRIBUTING.md]: Do not wrap lines.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
```
Comment (KaoImin) [inline CONTRIBUTING.md]: Do not wrap.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+### Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
```
Comment (KaoImin) [inline CONTRIBUTING.md]: Ditto

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at hello@nervos.org. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
```
Comment (KaoImin) [inline CONTRIBUTING.md]: Ditto

File: CONTRIBUTING.md (hunk 1 above)
Comment (mkxbl) [reply CONTRIBUTING.md]: This is conventional style

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+* Search existing issues to see whether others had already posted a similar issue.
+  
+* **Do not open up a GitHub issue to report security vulnerabilities**. Instead,
+  refer to the [security policy](SECURITY.md).
+
+* When creating a new issue, be sure to include a title and clear description. It is appreciated that if you can also attach as much relevant information as possible, such as version, environment, reproducing steps, samples.
+
+### Send PR
+
+* See [Code Standards]() for code guidelines.
```
Comment (yejiayu) [inline CONTRIBUTING.md]: Where are the links to `Code Standards`?

Comment (mkxbl) [reply CONTRIBUTING.md]: After code standards document is ready, i will fix that

**Pull request merged**